homeassistant/components/vizio/media_player.py

```diff
@@ -123,6 +150,7 @@ def __init__(
         self._state = None
         self._volume_level = None
         self._volume_step = volume_step
+        self._timeout = timeout
         self._current_input = None
         self._available_inputs = None
         self._device_type = device_type
```

Comment:
We're not using this instance attribute.

Instead we should implement the `device_class` property and return the device class there. So make an instance attribute `self._device_class`.